10/RequestReply.md

```diff
@@ -83,7 +83,37 @@ TODO understand the PUSH_BACK model. CORBA or more generic?
 
 #### Cache
 
-#### Serialisation
+#### Serialization
+
+The Serialization Mode defines how the Device Server handles multiple requests. 
+
+The Serialization SHALL apply to all requests independently of a request origin (network, the same machine, the same Device Server process). 
+
+A Device Server MAY handle requests with the following Serialization Modes:
+* Serialization by Device
+* Serialization by Class
+* Serialization by Process
+* No Serialization
+
+If Device Server is set to Serialization by Device:
+* it SHALL process requests to each Device in the order of requests arrival,
+* for any of Devices, it MUST NOT start the processing of a new request until the processing of the previous request is not finished,
+* it MAY process multiple requests in parallel, providing that there is maximum one processed request per a Device.
+
+If Device Server is set to Serialization by Class:
+* it SHALL process requests to each set of Devices belonging to the same Class in the order of requests arrival,
+* for each Device Class, it MUST NOT start the processing of a new request until the processing of the previous request is not finished,
+* it MAY process multiple requests in parallel, providing that there is maximum one processed request per Device Class.
+
+If Device Server is set to Serialization by Process:
+* it SHALL process requests in the order of requests arrival,
+* it MUST NOT start the processing of a new request until the processing of the previous request is not finished,
+
+The Device Server SHOULD process requests as soon as possible. If an incoming request cannot be processed due to Device Server 
+Serialisation Mode it SHOULD be queued for later processing and served as soon as possible.
+
+If Device Server is set to No Serialization it MAY process any requests in parallel. 
+It is NOT RECOMMENDED to use No Serialization mode. 
```

Comment:
I'd rephrase this, adding some explanation... what about:
"It is NOT RECOMMENDED to use No Serialization mode, unless the developer takes care of managing any concurrent access to the device data structures." or something similar...

Reply:
I'd also add:
 The default serialization Mode is "Serialization by Device".

Reply:
```suggestion
It is NOT RECOMMENDED to use No Serialization mode. 

The default serialization Mode is "Serialization by Device".
```